lib/src/main/java/net/ypresto/androidtranscoder/engine/MediaFormatValidator.java

```diff
@@ -36,7 +37,8 @@ public static void validateVideoOutputFormat(MediaFormat format) {
         }
         ByteBuffer spsBuffer = AvcCsdUtils.getSpsBuffer(format);
```

Comment:
Please remove lines below here, as profile validation is already not necessary.
https://github.com/ypresto/android-transcoder/pull/73#issuecomment-480259378

Reply:
@ypresto done! df7ee27